planning/scenario_selector/src/scenario_selector_node/scenario_selector_node.cpp

```diff
@@ -53,7 +53,7 @@ std::shared_ptr<lanelet::ConstPolygon3d> findNearestParkinglot(
 
   const auto linked_parking_lot = std::make_shared<lanelet::ConstPolygon3d>();
   const auto result = lanelet::utils::query::getLinkedParkingLot(
-    nearest_lanelet, all_parking_lots, linked_parking_lot.get());
+    search_point, all_parking_lots, linked_parking_lot.get());
```

Comment:
@NorahXiong Could you consider applying the following items? Or could I add commits to this PR?

- Remove `nearest_lanelet` and `nearest_lanelets` because they seem unused.
  - I'm not sure why it's not detected by `Wunused-variable`. :thinking: 
- Rename `search_point` to `current_position` to be easier to understand.